PLAT-7223: Remove replication failover mode. (#24)
memsql/S2-JDBC-Connector: README.md

    }
```

## Extended types support

Starting with SingleStore database version 8.7.1, extended types metadata is supported. 
This feature can be enabled using the `enable_extended_types_metadata` session variable. 
By default, this variable is set to false for backward compatibility, meaning the system will use the standard data type format.

With SingleStore JDBC driver version 1.2.6 and later, you can enable extended types metadata by setting the `enableExtendedDataTypes` parameter to true.
When enabled, JDBC will handle extended types metadata properly.

> **Note: The `enableExtendedDataTypes` JDBC parameter overrides the manually set `enable_extended_types_metadata` session variable.**

Reviewer: I don't understand what does this mean... Does it refer to the global variable `enable_extended_types_metadata`, or `SET SESSION enable_extended_types_metadata = [ON|OFF]` has no effect when called from the JDBC driver?

Author: means JDBC driver on connection establishment always execute query: SET SESSION enable_extended_types_metadata = true/false and will override any previously set.

Reviewer: okay, then let's replace "manually set `enable_extended_types_metadata` session variable." with "`enable_extended_types_metadata` server (global) variable"

    }
```

## Extended types support

> Starting with SingleStore database version 8.7.1, extended types metadata is supported. 

Reviewer 2: ```suggestion
SingleStore supports extended types metadata starting from database version 8.7.1. Enable the `enable_extended_types_metadata` session engine variable to use extended types in SingleStore. By default, support for extended types is disabled for backward compatibility and the database uses the standard data type format. 
```

This feature can be enabled using the `enable_extended_types_metadata` session variable. 

> By default, this variable is set to false for backward compatibility, meaning the system will use the standard data type format.

Reviewer 2: ```suggestion
```

With SingleStore JDBC driver version 1.2.6 and later, you can enable extended types metadata by setting the `enableExtendedDataTypes` parameter to true.

> Based on this parameter, the JDBC driver automatically sets the `enable_extended_types_metadata` session variable during connection initialization. 

Reviewer 2: ```suggestion
**Note**: Based on the value of the `enableExtendedDataTypes` parameter, the driver automatically configures the value of `enable_extended_types_metadata` engine variable while initializing the connection, which overrides the current value of this engine variable.
```

> When enabled, the JDBC driver will properly handle extended types metadata.

Reviewer 2: ```suggestion
```

> **Note: The JDBC driver always sets the `enable_extended_types_metadata` session variable to false or true during connection initialization, which may override any previously set value. By default, `enableExtendedDataTypes` is set to false.**

Reviewer 2: ```suggestion
```

Extended types:

> | Type                                                                                         | Affected changes                                                                                     |

Reviewer 2: ```suggestion
| Data Type                                                                                         | Updates                                                                                     |
```

|----------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|

> | [Vector](https://docs.singlestore.com/cloud/reference/sql-reference/data-types/vector-type/) | Added `com.singlestore.jdbc.type.Vector` type to read/write Vector data<br/>Updated metadata methods |

Reviewer 2: ```suggestion
| [Vector](https://docs.singlestore.com/cloud/reference/sql-reference/data-types/vector-type/) | Added `com.singlestore.jdbc.type.Vector` type to read/write Vector data.<br/>Updated metadata methods. |
```

> | [BSON](https://docs.singlestore.com/cloud/reference/sql-reference/data-types/bson-type/)     | Updated metadata methods                                                                             |

Reviewer 2: ```suggestion
| [BSON](https://docs.singlestore.com/cloud/reference/sql-reference/data-types/bson-type/)     | Updated metadata methods.                                                                             |
```
